Workflow/Modules/ModuleBase.py

```diff
@@ -357,7 +357,7 @@ def setApplicationStatus( self, status, sendFlag = True ):
       # The application status won't be updated in case the workflow or the step is failed already
       if not isinstance( status, basestring ):
         status = str( status )
-      self.log.verbose( 'setJobApplicationStatus(%d, %s)' % ( self.jobID, status ) )
+      self.log.verbose( 'setJobApplicationStatus(%s, %s)' % ( self.jobID, status ) )
```

Comment:
I don't know what's the issue here exactly, but definitely in this module `self.jobID` is printed several times as an `int`. And this line is pretty clear: https://github.com/DIRACGrid/DIRAC/blob/rel-v6r17/Workflow/Modules/ModuleBase.py#L109

I think you may just add in https://github.com/DIRACGrid/DIRAC/blob/rel-v6r17/Workflow/Modules/ModuleBase.py#L112
```
else:
  self.jobID = int(self.jobID)
```
to be sure.